### DISCUSSION TITLE
ID does not exist

### DISCUSSION THREAD

**Discussion by Jossyboydgenius** [rustdesk/rustdesk#10003] [General]
### Bug Description

When I try to connect using a remote ID, the app keeps showing a connection error; ID does not exist when I try on my Mac, Windows, and Android devices.

### How to Reproduce

Fix the error.

### Expected Behavior

To connect without issues.

### Operating system(s) on local (controlling) side and remote (controlled) side

Windows 10

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.2.3 -> 1.3.2

### Screenshots

<img width="1202" alt="Screenshot 2024-11-21 at 1 26 35 PM" src="https://github.com/user-attachments/assets/c03f7a36-27ec-44ce-91fd-dbe8a84b6b25">

![PHOTO-2024-11-21-13-31-57](https://github.com/user-attachments/assets/821a258a-fddb-4221-a353-fe836a1d1547)

![PHOTO-2024-11-21-13-34-00](https://github.com/user-attachments/assets/38638c1a-7f53-45eb-9b4a-18c04fa48826)

**Comment by sain1980** [comment]
For a few days now after Update to 1.3.6 I've been getting the message that the ID doesn't exist. I can only access one of the three PCs in my household. The others say "ID does not exist"
RustDesk version 1.3.6

Edit:
Somehow a relay server was entered in my local reinstallation and on the 2nd working PC. I deleted the entries and it works again on all PCs in the network.

the ralay server was :relay.rustdesk.io:21116

**Reply by open-trade** [reply]
https://github.com/rustdesk/rustdesk/discussions/9679

**Comment by reverrs** [comment]
Those who still consciously use a repeater server may also experience exactly the same error. This may be due, among other things, to the fact that all the necessary firewall ports were not opened during installation. In particular, TCP ports 21114 and UDP 21116 are often forgotten to open when using Yunohost and VDS with their own firewall, where the rules need to be written in two places. And of course, the full required range of TCP ports is 21114:21119.
